modules/debts_api/lib/debts_api/v0/fsr_form_transform/personal_data_calculator.rb

```diff
@@ -24,6 +23,14 @@ def get_personal_data
           }
         end
 
+        def name_str
+          name = ['veteran_full_name']
```

Comment:
**Question:** Why is `name` defined here? It's not used anywhere else in this method.

Reply:
outdated